packages/salesforcedx-vscode-lwc/src/messages/i18n.ts

```diff
@@ -59,5 +59,31 @@ export const messages = {
   watch_test_task_name: 'Watch Test',
   default_task_name: 'LWC Test',
   task_windows_command_prompt_messaging:
-    'Default shell for running tasks is set to cmd.exe'
+    'Default shell for running tasks is set to cmd.exe',
+  force_lightning_lwc_no_mobile_plugin:
```

Comment:
I think I would like to namespace the keys here related to mobile preview: `force_lightning_lwc_preview_mobile_no_mobile_plugin`, but that could be a refactor for later.